### PR TITLE
Substrate RC4

### PR DESCRIPTION
**Pull request by AurevoirXavier**

### CODE REVIEW

File: bin/node-template/runtime/src/lib.rs [hunk 1]
```diff
@@ -100,58 +105,6 @@ pub mod impls {
 		}
 	}
 
-	/// Update the given multiplier based on the following formula
```
Comment (HackFisher) [inline bin/node-template/runtime/src/lib.rs:103]: Did substrate have multiplier migration, changing from -1 to 0?

Comment (AurevoirXavier) [reply bin/node-template/runtime/src/lib.rs:103]: It's OK with Crab.

https://github.com/paritytech/substrate/pull/6334/files#diff-c624291b4dc7bad33847a05c04bdb83eR255

**Comment by HackFisher** [comment]
https://github.com/paritytech/substrate/pull/6334/files#diff-c624291b4dc7bad33847a05c04bdb83eR234

I worry that this migration codes will be removed in next releases(e.g. rc5), so need to keep an eye on this.

**Comment by AurevoirXavier** [comment]
> https://github.com/paritytech/substrate/pull/6334/files#diff-c624291b4dc7bad33847a05c04bdb83eR234
> 
> I worry that this migration codes will be removed in next releases(e.g. rc5), so need to keep an eye on this.

https://github.com/darwinia-network/darwinia/issues/478

**Pull request merged**